src/containers/my-certificates/certificate-table/certificate-table.js

```diff
@@ -19,20 +20,22 @@ const CertificateTable = ({ items, openModal }) => {
   );
 
   const headerItems = useMemo(
```

Comment:
Header and body items should be handled inside of table component.

Reply:
Body items render component with many props, so i need to pass all this props to table from certificateTable and OrderTable component and also check if it is table for certificate or for cart.